### PR TITLE
app-emulation/genymotion-bin-3.0.4 version bump

### PR DESCRIPTION
**Pull request by fedeliallalinea**

### CODE REVIEW

File: app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+DESCRIPTION="Complete set of tools that provide a virtual environment for Android"
+HOMEPAGE="https://genymotion.com"
+SRC_URI="${BIN_ARCHIVE}"
+
+LICENSE="genymotion"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+DEPEND=""
```
Comment (juippis) [inline app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild]: Empty variable can be dropped.

File: app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2019 Gentoo Authors
```
Comment (juippis) [inline app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild]: Please also update the copyright year now (I know it was committed in 2019)

File: app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+DESCRIPTION="Complete set of tools that provide a virtual environment for Android"
+HOMEPAGE="https://genymotion.com"
+SRC_URI="${BIN_ARCHIVE}"
+
+LICENSE="genymotion"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+DEPEND=""
+RDEPEND="|| ( >=app-emulation/virtualbox-5.0.28 >=app-emulation/virtualbox-bin-5.0.28 )
```
Comment (juippis) [inline app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild]: Multiline this, version restriction can be removed as it's always satisfied in ::gentoo.

File: app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+	virtual/opengl
```
Comment (juippis) [inline app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild]: Put this below everything to sort this list nicely.

File: app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+	|| (
+		dev-libs/openssl-compat:1.0.0
+		=dev-libs/openssl-1.0*:0
+	)
+	dev-libs/hiredis:0/0.14
+	sys-apps/util-linux
+"
+BDEPEND=">=dev-util/patchelf-0.9_p20180129"
```
Comment (juippis) [inline app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild]: Version is always satisfied in ::gentoo, can be cleaned out.

File: app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+	for i in genymotion genyshell gmtool player libcom.so.1.0.0 librendering.so.1.0.0 ; do
+		patchelf --replace-needed libhiredis.so.0.13 libhiredis.so.0.14 "${MY_PN}/${i}" || die "Unable to patch ${i} for hiredis"
+	done
+}
+
+src_install() {
+	insinto /opt/"${MY_PN}"
+	exeinto /opt/"${MY_PN}"
+
+	# Use qt bundled
```
Comment (juippis) [inline app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild]: Any chance to use system-Qt?

Comment (fedeliallalinea) [reply app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild]: Unfortunately not, if I remove all qt bundled library and I run program I have following error:
`[Genymotion Shell:18945] [fatal] Cannot mix incompatible Qt library (version 0x50906) with this library (version 0x50e01)`
this means the program is compiled with qt 5.9.6

**Pull request closed without merging**